src/test/resources/application.conf

```diff
@@ -115,9 +115,8 @@ mantis {
     block-bodies-per-request = 10
     receipts-per-request = 10
     nodes-per-request = 10
-    min-peers-to-choose-pivot-block = 2
-    peers-to-choose-pivot-block-margin = 1
-    pivot-block-offset = 500
+    min-peers-to-choose-target-block = 2
+    target-block-offset = 500
```

Comment:
Why did you make this change? I think it is incorrect. `min-peers-to-choose-target-block` and `target-block-offset` are not used anymore